pkg/environment/environment.go

```diff
@@ -59,6 +59,16 @@ func (p Paths) BasePath() string { return p.base }
 // e.g. {IndexPath}/plugins/{plugin}.yaml
 func (p Paths) IndexPath() string { return filepath.Join(p.base, "index") }
 
+// IndexPluginsPath returns the plugins directory of the index repository.
+//
+// e.g. {IndexPath}/plugins/
+func (p Paths) IndexPluginsPath() string { return filepath.Join(p.base, "index", "plugins") }
+
+// ReceiptsPath returns the base directory where plugin receipts are stored.
+//
+// e.g. {ReceiptsPath}/krew-index/{plugin}.yaml
+func (p Paths) ReceiptsPath() string { return filepath.Join(p.base, "receipts", "krew-index") }
```

Comment:
re `krew-index/` segment: I think you were planning on multi-index migration ahead of time, which is great. But I think a flat list of plugins should be saved inside `receipts/` in this case. Here's why:

We won't allow installing plugins `indexA/foo` and `indexB/foo` at the same time, so there will be just only `foo` installed at a time. One way to ensure that sounds like we could just do `receipts/foo.yaml` instead of  adding `receipts/index-name/...` hierarchy.

It's open for discussion, but my hunch is that this shouldn't include the index-name.
For example when you do `krew info foo`, we could just look up `receipts/foo.yaml` (regardless of the index).
But this brings up an issue: how do you tell "which index this plugin came from". Maybe that can be resolved with a `status:` field. I don't know yet.